R/produce_sc_ch_episodes_tests.R

```diff
@@ -25,5 +25,5 @@ produce_sc_ch_episodes_tests <- function(data) {
       .data$ch_nursing, .data$ch_adm_reason, .data$sc_latest_submission
     )) %>%
     # use function to sum new test flags
-    sum_test_flags()
+    calculate_measures(measure = "sum")
```

Comment:
Tested this and it works okay. If the vars are NULL and the measure is sum then this will summarise for test flags. If we want the sum and mean then this is specified as all but will need to specify variables which will match e.g. 'beddays', 'cost'